examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Change the title from "OQC Lucy" to "OQC's Lucy for consistency with <a href="https://github.com/amazon-braket/amazon-braket-examples/blob/main/examples/pulse_control/3_Bell_pair_with_pulses_OQC.ipynb" target="_blank">https://github.com/amazon-braket/amazon-braket-examples/blob/main/examples/pulse_control/3_Bell_pair_with_pulses_OQC.ipynb</a></p><p> </p><p>The last paragraph is great but is missing a plot of a bloch sphere or at least a reference. Would be fine to say "see plots below".</p><p> </p><p>"To access remote services on Amazon Braket, you first need to <a href="https://aws.amazon.com/braket/getting-started/" target="_blank">create an &nbsp;account on AWS</a>&nbsp;and&nbsp;follow the <a href="https://github.com/aws/amazon-braket-sdk-python" target="_blank">setup &nbsp;instructions</a>&nbsp;for&nbsp;accessing Braket from Python. You also need to install the &nbsp;<a href="https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/" target="_blank">pennylane-braket &nbsp;plugin</a>." we generally do not include this in our notebook examples. Please remove. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
✅